### PR TITLE
Added basic stress-arousal system

### PR DESCRIPTION
**Pull request by it-thatlikeswarmdrinks**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change. Likely not save-compatible)
- [x] My change requires a change to the documentation/wiki.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: events/carn_arousal_events.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	}
+	immediate = {
+		send_interface_message = {
+			title = carn_arousal.0001.t
+			#desc = carn_arousal.0001.desc
+
+			add_stress = {
+				# Using -1 * medium_stress_loss as the base value instead of something more straightforward so as to balance the effect of carn_sex_interaction.
+				value = medium_stress_loss 	
+				multiply = -1
```
Comment (Triskelia) [inline events/carn_arousal_events.txt:25]: I don't see why we would not use "medium_value_gain" here

Comment (it-thatlikeswarmdrinks) [reply events/carn_arousal_events.txt:25]: medium_stress_loss = 20 by default but medium_stress_gain = 40. I was thinking that this way the stress gain for a non-lustful character would exactly balance the loss from Lay With even if the user has installed some kind of "stress overhaul" mod that changes the scripted values.

Comment (Triskelia) [reply events/carn_arousal_events.txt:25]: Ok, make sense!
I leave it to Cheri to accept the pull request since she has a better understanding of the framework than me
